static/css/restyle.css

```diff
@@ -0,0 +1,695 @@
+/* Hide outlines Firefox shows on every click */
+* {
+  outline: none;
```

Comment:
Yeah we had some discussion about this and I was going to bring it up. The outline is pretty gross looking in the UI and it is unfortunate that's the case here. I can remove it, I don't think any of the other suggestions for fixing it really applies.